spring-cql/src/main/java/org/springframework/cassandra/core/converter/RowToListConverter.java

```diff
@@ -23,7 +24,8 @@
 
 		for (Definition def : cols.asList()) {
 			String name = def.getName();
-			list.add(row.isNull(name) ? null : def.getType().deserialize(row.getBytesUnsafe(name)));
+			list.add(row.isNull(name) ? null : def.getType().deserialize(
+					row.getBytesUnsafe(name), ProtocolVersion.NEWEST_SUPPORTED));
```

Comment:
Therefore this method has to know one of the objects `Cluster`, `Configuration`, `ProtocolOptions` or `ProtocolVersion` itself. This cannot be done without changing the method's signature or the class' constructor. In think this is the reason for using `ProtocolVersion.NEWEST_SUPPORTED` in commit e1a197170b94cf4df93d5b25c4db43a0c7f63282, too. I suggest to merge this commit like it is in order to avoid the exception. Replacing the `ProtocolVersion.NEWEST_SUPPORTED` should be a new issue that considers the other classes that use `ProtocolVersion`, too.